generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<p>It's not clear what the manipulated images are versus the originals.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Modified caption and commented results. </p>